scripts/regenerate-protobuf-output.sh

```diff
@@ -21,3 +22,11 @@ if ! GO_MOD_FILEPATH="${root_dirpath}/${GOLANG_DIRNAME}/go.mod" "${GENERATOR_SCR
     exit 1
 fi
 echo "Successfully generated Go bindings in directory '${go_output_dirpath}'"
+
+#TypeScript
+typescript_output_dirpath="${root_dirpath}/${TYPESCRIPT_DIRNAME}/core_api_bindings"
```

Comment:
I suspect you'll end up changing the eventual output directory once you understand Typescript packaging systems more, but this is fine for now